test/integration/integration_test.go

```diff
@@ -55,16 +74,14 @@ func testBasicConfig(grpcPort, perSecond string) func(*testing.T) {
 		os.Setenv("RUNTIME_ROOT", "runtime/current")
 		os.Setenv("RUNTIME_SUBDIRECTORY", "ratelimit")
 		os.Setenv("REDIS_PERSECOND_SOCKET_TYPE", "tcp")
-		os.Setenv("REDIS_PERSECOND_URL", "localhost:6380")
 		os.Setenv("REDIS_SOCKET_TYPE", "tcp")
-		os.Setenv("REDIS_URL", "localhost:6379")
 
 		go func() {
 			runner.Run()
 		}()
 
 		// HACK: Wait for the server to come up. Make a hook that we can wait on.
-		time.Sleep(100 * time.Millisecond)
+		time.Sleep(100 * time.Second)
```

Comment:
why is the waiting time longer?

Reply:
I needed to increase this because of the additional redis servers, and stunnel that are spun up in the travis build. I've made this a more sane value at 1s instead of 100s.